newton/_src/viewer/viewer.py

```diff
@@ -94,6 +97,62 @@ def set_model(self, model):
     def set_camera(self, pos: wp.vec3, pitch: float, yaw: float):
         pass
 
+    def set_world_offsets(self, num_worlds: int, spacing: tuple[float, float, float] = (5.0, 5.0, 0.0)):
+        """Set world offsets for visual separation of multiple worlds.
+
+        Args:
+            num_worlds: Number of worlds
+            spacing: Spacing between worlds along each axis (x, y, z)
+        """
+        # Compute offsets using the same logic as ModelBuilder._compute_replicate_offsets
```

Comment:
Can you put this function somewhere outside the ModelBuilder, maybe in the utils module? Then you can reuse this.